javascript/javascript3/week1/giphy/giphy.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <input
+      type="text"
+      name="searchGif"
+      id="search-gif"
+      placeholder="search for gif"
+    />
+    <br />
+    <input type="text" id="gifs-number" placeholder="How many gifs" />
+    <br />
+    <button id="gif-btn">Search</button>
+    <div class="output"></div>
```

Comment:
We should use `id` here as well to identify the `div`. `class` is better used when we want to affect the same elements in the same way like through css.